pkg/apis/secret/v0alpha1/secure_value.go

```diff
@@ -46,6 +46,9 @@ type SecureValueStatus struct {
 	// Only applicable if the `phase=Failed`.
 	// +optional
 	Message string `json:"message,omitempty"`
+
+	// +optional
+	ExternalID string `json:"externalId,omitempty"`
```

Comment:
It is a bit weird that this is inside the `status` and not the `spec` 🤔 ? Any particular reason to put it here?

Reply:
Fair enough!